src/views/FaceContent.vue

```diff
@@ -0,0 +1,478 @@
+<!--
+ - @copyright Copyright (c) 2022 Louis Chemineau <louis@chmn.me>
+ - @copyright Copyright (c) 2022 Marcel Klehr <mklehr@gmx.net>
+ -
+ - @author Louis Chemineau <louis@chmn.me>
+ - @author Marcel Klehr <mklehr@gmx.net>
+ -
+ - @license AGPL-3.0-or-later
+ -
+ - This program is free software: you can redistribute it and/or modify
+ - it under the terms of the GNU Affero General Public License as
+ - published by the Free Software Foundation, either version 3 of the
+ - License, or (at your option) any later version.
+ -
+ - This program is distributed in the hope that it will be useful,
+ - but WITHOUT ANY WARRANTY; without even the implied warranty of
+ - MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE. See the
+ - GNU Affero General Public License for more details.
+ -
+ - You should have received a copy of the GNU Affero General Public License
+ - along with this program. If not, see <http://www.gnu.org/licenses/>.
+ -
+ -->
+<template>
+	<!-- Errors handlers-->
+	<EmptyContent v-if="face === undefined && !loadingFiles && !loadingFaces" class="empty-content-with-illustration">
+		<template #icon>
+			<AccountBoxMultipleOutline />
+		</template>
+		{{ t('photos', 'This person could not be found') }}
+	</EmptyContent>
+	<EmptyContent v-else-if="errorFetchingFiles || errorFetchingFaces">
+		<template #icon>
+			<AlertCircle />
+		</template>
+		{{ t('photos', 'An error occurred') }}
+	</EmptyContent>
+
+	<div v-else class="album">
```

Comment:
also face not album.

I'll probably abstract a bunch of the code for one of the photos2 next task anyway, so those tiny error won't stay for long :)